web/styles/animations.sass

```diff
@@ -34,6 +34,7 @@ $transition-time-long: 1.5s
     width: 100%
     z-index: -100
     -webkit-transition: $transition
-  &:hover
-     &:before
-       opacity: 1 
+  @media (hover:hover)
```

Comment:
does this have something to do with making this work with images?

Reply:
nope http://www.javascriptkit.com/dhtmltutors/sticky-hover-issue-solutions.shtml
I think in mobile when you click something it stays in `hover` mode and this is one way to deal with it